### PR TITLE
feat(impact analysis): bugfixes for Impact Analysis

### PR DESCRIPTION
**Pull request by gabe-lyons**
Addresses:

1) arrow keys not working on download modal
2) adds tracking
3) fixes search layout nit
4) usernames & emails not appearing

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: datahub-web-react/src/app/entity/shared/components/styled/search/downloadAsCsvUtil.ts [hunk 1]
```diff
@@ -48,7 +48,12 @@ export const transformGenericEntityPropertiesToCsvRow = (
         // user owners
         properties?.ownership?.owners
             ?.filter((owner) => owner.owner.type === EntityType.CorpUser)
-            .map((owner) => (owner.owner as CorpUser).properties?.fullName)
+            .map(
+                (owner) =>
+                    (owner.owner as CorpUser).properties?.fullName ||
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/entity/shared/components/styled/search/downloadAsCsvUtil.ts:53]: Actually i feel the order should be reversed maybe? 

If they've set a display name, use that. If not, fallback to other information

File: datahub-web-react/src/app/entity/shared/components/styled/search/DownloadAsCsvModal.tsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                        Download
+                    </Button>
+                </>
+            }
+        >
+            <Input
+                placeholder="datahub.csv"
+                value={saveAsTitle}
+                onChange={(e) => {
+                    console.log(e);
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/entity/shared/components/styled/search/DownloadAsCsvModal.tsx:118]: prob don't need this!

**Review by jjoyce0510 — APPROVED** [review]
minor comments, else lgtm

**Comment by gabe-lyons** [comment]
@jjoyce0510 addressed!

**Review by jjoyce0510 — APPROVED** [review]

**Review by shirshanka — APPROVED** [review]

**Pull request merged**